### PR TITLE
fix: product cards with SfLink inside slots

### PR DESCRIPTION
**Pull request by martaradziszewska**
# Scope of work

- [x]  move `SfLink` to slots

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [x] Run tests and docs
- [x] Self code-reviewed

If applicable:

- [ ] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [ ] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))
 - [ ] I accept [Individual Contributor License Agreement](https://github.com/DivanteLtd/next/blob/master/CLA.md)

### CODE REVIEW

File: packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue [hunk 1]
```diff
@@ -1,7 +1,11 @@
 <template>
   <div class="sf-product-card">
     <div class="sf-product-card__image-wrapper">
-      <SfLink class="sf-product-card__link" :link="link">
+      <component
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue:4]: @martaradziszewska maybe we can just move SfLink with an image to slot? It should be easier to replaced? WDYT?

Comment (martaradziszewska) [reply packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue]: that is also a good idea, I'll try this :)

File: packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <template>
   <div class="sf-product-card">
     <div class="sf-product-card__image-wrapper">
-      <SfLink class="sf-product-card__link" :link="link">
-        <slot name="image" v-bind="{ image, title }">
+      <slot name="image" v-bind="{ image, title }">
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue:4]:
```suggestion
      <slot name="image" v-bind="{ image, title, link, imageWidth, imageHeight }">
```

File: packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue [hunk 3]
```diff
@@ -76,13 +76,13 @@
         </slot>
       </template>
     </div>
-    <SfLink class="sf-product-card__link" :link="link">
-      <slot name="title" v-bind="{ title }">
+    <slot name="title" v-bind="{ title }">
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue:79]:
```suggestion
    <slot name="title" v-bind="{ title, link }">
```

File: packages/vue/src/components/organisms/SfProductCardHorizontal/SfProductCardHorizontal.vue [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-    <SfLink
-      :link="link"
-      class="sf-product-card-horizontal__link sf-product-card-horizontal__link--image"
-    >
-      <div class="sf-product-card-horizontal__image-wrapper">
-        <!--@slot Use this slot to replace image-->
-        <slot name="image" v-bind="{ image, title }">
+    <div class="sf-product-card-horizontal__image-wrapper">
+      <!--@slot Use this slot to replace image-->
+      <slot name="image" v-bind="{ image, title }">
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfProductCardHorizontal/SfProductCardHorizontal.vue:5]:
```suggestion
      <slot name="image" v-bind="{ image, imageWidth, imageHeight, title, link }">
```

File: packages/vue/src/components/organisms/SfProductCardHorizontal/SfProductCardHorizontal.vue [hunk 5]
```diff
@@ -26,19 +26,19 @@
...
+        </SfLink>
+      </slot>
+    </div>
     <div class="sf-product-card-horizontal__main">
       <div class="sf-product-card-horizontal__details">
-        <SfLink :link="link" class="sf-product-card-horizontal__link">
-          <!--@slot Use this slot to replace title-->
-          <slot name="title" v-bind="{ title }">
+        <!--@slot Use this slot to replace title-->
+        <slot name="title" v-bind="{ title }">
```
Comment (pspaczek) [inline packages/vue/src/components/organisms/SfProductCardHorizontal/SfProductCardHorizontal.vue:35]:
```suggestion
        <slot name="title" v-bind="{ title, link }">
```

**Pull request merged**